exporter/otlpexporter/otlp.go

```diff
@@ -36,26 +36,27 @@ import (
 
 type exporter struct {
 	// Input configuration.
-	config *Config
-	w      *grpcSender
+	config    *Config
+	w         *grpcSender
+	buildInfo component.BuildInfo
```

Comment:
I wonder if we should put `buildInfo` in the TelemetrySettings, any thoughts @bogdandrutu?

Reply:
How would that improve things?